projects/1.16/assets/xaeros-minimap/xaerobetterpvp/lang/zh_cn.json

```
@@ -1,354 +1,457 @@
+  "gui.xaero_add_edit": "添加/编辑",
+  "gui.xaero_new_waypoint": "新建路径点",
+  "gui.xaero_initial": "首字母",
   "gui.xaero_edit_waypoint": "编辑路径点",
-  "gui.xaero_initial": "初始化",
-  "gui.xaero_new_waypoint": "新的路径点",
-  "gui.xaero_edit_mode": "编辑模式",
-  "gui.xaero_miscellaneous": "杂项",
+  "gui.xaero_armour_status_settings": "盔甲状态设置",
+  "gui.xaero_choose_a_preset": "选择一个预设",
+  "gui.xaero_choose_an_interface": "选择一个界面",
+  "gui.xaero_armour_status": "盔甲状态",
+  "gui.xaero_notifications": "通知",
   "gui.xaero_minimap": "小地图",
-  "gui.xaero_add_edit": "添加/编辑",
-  "gui.xaero_delete": "删除",
-  "gui.xaero_back": "返回",
-  "gui.xaero_cancel": "取消",
-  "gui.xaero_confirm": "确认",
-  "gui.xaero_hide_wp_coords": "隐藏路径点图标",
-  "gui.xaero_arrow": "箭头",
-  "gui.xaero_compass": "指南针",
-  "gui.xaero_box_entity_nametags": "对于具有名称标记的实体，在点下显示实体名称。",
-  "gui.xaero_potion_effects_blink": "低时闪烁",
+  "gui.xaero_miscellaneous": "杂项",
+  "gui.xaero_edit_mode": "编辑模式",
+  "gui.xaero_instructions": "帮助",
+  "gui.xaero_not_ingame": "你必须处于游戏中才能使用编辑模式！",
+  "gui.xaero_welcome_edit_mode": "欢迎进入编辑模式！",
+  "gui.xaero_howto_select": "左键选择一个界面。",
+  "gui.xaero_howto_drag": "左键拖动来移动界面。",
+  "gui.xaero_howto_deselect": "右键来取消选择一个界面。",
+  "gui.xaero_howto_center": "按下“C”来居中或取消居中一个界面。",
+  "gui.xaero_different_centered": "部分界面有着不同的居中样式。",
+  "gui.xaero_howto_flip": "按下“F”来翻转或取消翻转一个界面。",
+  "gui.xaero_howto_settings": "按下“S”来快速访问所选界面的设置。",
+  "gui.xaero_howto_preset": "点击“选择一个预设”按钮来选择一个预设。",
+  "gui.xaero_howto_save": "点击“确认”按钮来保存你的更改。",
+  "gui.xaero_howto_cancel": "点击“取消”按钮来取消你的更改。",
+  "gui.xaero_minimap_settings": "小地图设置",
+  "gui.xaero_miscellaneous_settings": "杂项设置",
+  "gui.xaero_notification_settings": "通知设置",
+  "gui.xaero_better_pvp_settings": "Better PvP 设置",
+  "gui.xaero_server_disabled": "服务器可能禁用了此 mod 的一些功能。",
+  "gui.xaero_waypoints": "路径点",
+  "gui.xaero_box_potion_effects": "一个显示当前激活的药水效果的界面。",
+  "gui.xaero_box_armour_status": "一个显示当前装备和主手物品信息的界面。",
+  "gui.xaero_box_sprint": "使用一个单独的按键（可配置的）来冲刺。",
+  "gui.xaero_box_sneak": "允许你切换潜行状态，并且在潜行状态中不需要按住任何按键。",
+  "gui.xaero_box_notifications": "为一些信息启用屏幕通知，如生命值低、饱食度低、氧气量低、苦力怕预警、或者骷髅怪预警。",
+  "gui.xaero_box_xp": "获得或者失去经验值时将会有提示。",
+  "gui.xaero_box_quick_use": "一个类似于动作条的热键系统，可以让你在不将主手物品换掉的情况下使用指定的物品。按住热键来持续使用物品，就像按住右键时一样。比如：放置火把、TNT、扔药水、吃东西、喝东西等等。",
+  "gui.xaero_box_minimap": "一个显示你周围环境与实体的小地图。",
+  "gui.xaero_centered": "已居中:",
+  "gui.xaero_press_c": "(按下 C)",
+  "gui.xaero_flipped": "已翻转:",
+  "gui.xaero_press_f": "(按下 F)",
+  "gui.xaero_preset_fancy": "幻想",
+  "gui.xaero_preset_classic": "经典",
+  "gui.xaero_preset_social": "社交",
+  "gui.xaero_potion_status": "药水效果",
+  "gui.xaero_archery_status": "显示弓箭剩余数量",
+  "gui.xaero_sprint": "更好的冲刺",
+  "gui.xaero_sneak": "切换潜行",
+  "gui.xaero_enchants_color": "附魔颜色",
+  "gui.xaero_durability": "耐久度",
+  "gui.xaero_hunger_setting": "饱食度低提醒",
+  "gui.xaero_hp_setting": "健康值低提醒",
+  "gui.xaero_explosion_setting": "苦力怕预警",
+  "gui.xaero_being_shot_setting": "骷髅怪预警",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "gui.xaero_being_shot_setting": "骷髅预警",
```